client/src/components/generic/Page.js

```diff
@@ -14,13 +14,24 @@ const useStyles = makeStyles(() => ({
   }),
 }));
 
+const useStyleAutoHeight = makeStyles(() => ({
+  root: (props) => ({
+    height: 'auto',
+    justifyContent: props.centered ? 'center' : 'flex-start',
+    alignItems: 'center',
+    flexWrap: 'nowrap',
+    flexDirection: 'column',
+  }),
+}));
+
 // hideEmployers set to true for participant-facing pages
-export const Page = ({ children, hideEmployers = false, centered }) => {
+export const Page = ({ children, hideEmployers = false, centered, isAutoHeight = false }) => {
   const classes = useStyles({ centered });
+  const classesAuto = useStyleAutoHeight({ centered });
```

Comment:
It feels a bit weird to have multiple 'makeStyles' and switching the root style base on a prop. Normally I would have only one with multiple classes within it, and switch classes based on the prop.

Reply:
Yes, that is right. I will update date that.